### PR TITLE
Add figures for fill characters

### LINKED ISSUE #57: Add figures for fill characters
What are your thoughts on adding figures for fill characters? 

They display well on all environments. The following screenshots shows them both as individual characters, and displayed on several rows/columns.

Ubuntu 20.10, Gnome terminal:

unix_1

Windows 10, `cmd.exe` (CP850):

windows_1

On macOS default terminal, they do not combine perfectly vertically. I do not think there is a better character available though (but please let me know if I'm wrong). Also, many macOS users might have better terminal / fonts / settings which would show this better.

macos_1

I believe all of the above are useful:
  - The first 4 fill the whole character, which is good when "painting" a whole area, with 4 different shades
  - `U-25a0` is useful for progress bar, as it has margins both up and down, but not left and right
  - The other 4 are useful as half-full characters: left, right, up, down

### PR DESCRIPTION
**Pull request by ehmicky**
Fixes #57.

This adds some figures for fill characters.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -5,6 +5,14 @@ declare const figureSet: {
 	readonly square: string;
 	readonly squareSmall: string;
 	readonly squareSmallFilled: string;
+	readonly squareLight: string;
+	readonly squareMedium: string;
+	readonly squareDark: string;
```
Comment (ehmicky) [inline index.d.ts:10]: Whether those squares are light or dark depends on the terminal background color.
The above assumes the terminal background color is dark, which I believe is the most common. The official Unicode codepoint names assumes the opposite.

Another option would be to rename them in a way that does not imply brightness. However, I could not find any elegant way to do so.

File: index.d.ts [hunk 2]
```diff
@@ -5,6 +5,14 @@ declare const figureSet: {
...
+	readonly squareTop: string;
+	readonly squareBottom: string;
+	readonly squareLeft: string;
+	readonly squareRight: string;
+	readonly squareCenter: string;
```
Comment (ehmicky) [inline index.d.ts:15]: `squareCenter` has margins on top, bottom but not on left and right. I am not sure whether this name conveys this properly.

File: index.d.ts (hunk 1 above)
Comment (sindresorhus) [reply index.d.ts:10]: The official name is "light shade". I would add the `shade` prefix to make it clear it's not about the color, but the opacity.

Comment (ehmicky) [reply index.d.ts]: Fixed in 41c7da9

**Comment by sindresorhus** [comment]
Can you fix the conflict?

**Comment by ehmicky** [comment]
Fixed :+1:

**Pull request merged**